packages/opentelemetry-api/package.json

```diff
@@ -10,18 +10,17 @@
   },
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "compile": "tsc --build",
-    "clean": "tsc --build --clean",
-    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "compile": "../../node_modules/.bin/tsc --build",
```

Comment:
how is this going to work if you install later the package without lerna in your final project and it will try to compile it using the node_modules folder that doesn't exist ?

Reply:
If you install the API as a dependency of your project you already have the built js files there is no need to run tsc. The tsconfig.json and other related files are not even included in the tgz file built by `npm pack`.